shell/browser/auto_updater.h

```diff
@@ -55,7 +58,10 @@ class AutoUpdater {
   static void SetDelegate(Delegate* delegate);
 
   static std::string GetFeedURL();
-  static void SetFeedURL(mate::Arguments* args);
+  // FIXME(zcbenz): We should not do V8 in this file, this method should only
+  // accept C++ struct as parameter, and atom_api_auto_updater.cc is responsible
+  // for parsing the parameter from JavaScript.
+  static void SetFeedURL(gin_helper::Arguments* args);
```

Comment:
Since this is `static` the refactor wouldn't be much and would be good to do it in this PR ?

Reply:
I would like to keep this PR focused on native_mate related changes.